### PR TITLE
Parse out system Integration, create Linux integration

### PR DESCRIPTION
**Pull request by fearful-symmetry**
## What does this PR do?

This is in preparation for 7.10. We want to strip the system integration down to just cross-compatible datasets. This will be the first of a few changes for 7.10, the rest will involve cleaning up the existing cross-compatible datasets.

## Checklist

- [X] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/master/CONTRIBUTING.md#tips-for-building-integrations) and this pull request is aligned with them.
- [X] I have verified that all datasets collect metrics or logs.

## Author's Checklist

- [ ] Do we want a custom icon for the Linux Integration?

## How to test this PR locally

- Pull down the PR
- Build the new package registry
- Enroll a test agent, make sure the `linux` and `system` integrations are properly ingesting data

### CODE REVIEW

**Comment by mtojek** [comment]
Please run `mage check` locally as some files seem to be not formatted well.

**Comment by mtojek** [comment]
```
[2020-09-23T14:23:32.837Z] packages/linux/docs/README.md: needs update
[2020-09-23T14:23:32.837Z] Error: running "git update-index --refresh" failed with exit code 1
```

Please refresh docs using `mage GenerateDocs` (also part of `mage check`).

**Comment by fearful-symmetry** [comment]
Ah, there we go!

File: dev/import-beats-resources/linux/docs/README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+It collects the amount of available entropy in bits. On kernel versions greater than 2.6, 
+entropy will be out of a total pool size of 4096.
+
+{{fields "entropy"}}
+
+### Network summary
+
+The System `network_summary` dataset provides network IO metrics collected from the
+operating system. These events are global and sorted by protocol.
+
```
Comment (mtojek) [inline dev/import-beats-resources/linux/docs/README.md:26]: nit: empty line

File: packages/linux/manifest.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    size: 5006x2260
+    type: image/png
+config_templates:
+  - name: system
+    title: Linux kernel metrics
+    description: Collect system metrics from Linux operating systems
+    inputs:
+      - type: system/metrics
+        title: Collect metrics from Linux instances
+        description: Collecting System entropy, network_summary, raid, service, socket, and users metrics
```
Comment (mtojek) [inline packages/linux/manifest.yml:25]: nit: you can change "network_summary" to "network summary" for ease of reading
nit: RAID (also please check other places).

**Review by mtojek — APPROVED** [review]
>  Do we want a custom icon for the Linux Integration?

I suppose that Tux is a good candidate here.

Overall, LGTM!

**Pull request merged**